UI/frontend-plugins/frontend-tools/scripts.cpp

```diff
@@ -190,15 +192,30 @@ ScriptsTool::ScriptsTool() : QWidget(nullptr), ui(new Ui_ScriptsTool)
 	RefreshLists();
 
 #if PYTHON_UI
-	config_t *config = obs_frontend_get_global_config();
-	const char *path =
-		config_get_string(config, "Python", "Path" ARCH_NAME);
-	ui->pythonPath->setText(path);
-	ui->pythonPathLabel->setText(obs_module_text(PYTHONPATH_LABEL_TEXT));
-#else
-	delete ui->pythonSettingsTab;
-	ui->pythonSettingsTab = nullptr;
-	ui->tabWidget->setStyleSheet("QTabWidget::pane {border: 0;}");
+	if (!obs_scripting_python_loaded()) {
+		config_t *global_config = obs_frontend_get_global_config();
+
+		if (!config_get_bool(global_config, "scripts-tool",
+				     "hidePythonWarning")) {
+			QString message =
+				QT_UTF8(obs_module_text("PythonWarning"))
+					.arg(PYTHON_VERSION);
+
+			QMessageBox mb(this);
+			mb.setIcon(QMessageBox::Warning);
+			mb.setWindowTitle(
+				obs_module_text("PythonNotInstalled"));
+			mb.setText(message);
+
+			QCheckBox *cb = new QCheckBox();
+			cb->setText(obs_module_text("DoNotShowAgain"));
```

Comment:
Rather than a duplicate `DoNotShowAgain` translation key, Jim taught me this trick.

```cpp
obs_frontend_push_ui_translation(obs_module_get_string);
cb->setText(tr("DoNotShowAgain"));
obs_frontend_pop_ui_translation();
```

This'll pull it from the UI locale.

Reply:
Done.